src/component/UserProfile/UserRegister.tsx

```diff
@@ -377,9 +453,32 @@ export function UserRegister(props: userRegisterProps) {
           helperText={passwordErrorTxt}
         />
 
+          <FormControl sx={{ width: '100%' }}>
+              <InputLabel id="community-multiple-checkbox-label">Watch Group</InputLabel>
+              <Select
+                  labelId="community-multiple-checkbox-label"
+                  id="community-multiple-checkbox"
+                  multiple
+                  value={selectedCommunities}
+                  onChange={handleCommunityChange}
+                  input={<OutlinedInput label="Tag" />}
+                  renderValue={(selected) => selected.join(', ')}
+                  MenuProps={CommunitySelectProps}
+                  error={!communityValid}
+              >
+                  {availCommunities.map((community) => (
+                      <MenuItem key={community.id} value={community.name}>
+                          <Checkbox checked={selectedCommunities.indexOf(community.name) > -1} />
+                          <ListItemText primary={community.name} />
+                      </MenuItem>
+                  ))}
+              </Select>
+            <FormHelperText sx={{ color: '#e74c3c' }} hidden={communityValid}>{communityErrorTxt}</FormHelperText>
+          </FormControl>
+
         <Box
           component='span'
-          sx={{ m: 1, color: 'red' }}
+          sx={{ m: 1, color: '#e74c3c' }}
```

Comment:
Color is different than all the other error fields. This is what we're using with material currently. 
.css-1wc848c-MuiFormHelperText-root.Mui-error {
    **color: #d32f2f;**
}

Reply:
huh, thats what i pulled from the inspect

Reply:
![image](https://github.com/Lean-Mean-Coding-Machines/MovieNightManagerPortal/assets/54011905/2ba643ce-ece2-460d-8e53-de429e6e86cf)

It's slightly lighter than the rest. You likely pulled that from the old error message that displayed above the submit btn.